undertow/src/main/java/org/wildfly/extension/undertow/deployment/UndertowDeploymentProcessor.java

```diff
@@ -408,14 +417,31 @@ public Duration getDefaultSessionTimeout() {
             };
             CapabilityServiceConfigurator factoryConfigurator = provider.getSessionManagerFactoryServiceConfigurator(factoryServiceName, configuration);
             CapabilityServiceConfigurator codecConfigurator = provider.getSessionIdentifierCodecServiceConfigurator(codecServiceName, configuration);
```

Comment:
We only want to install this if the ServletContainerService does *not* contain affinity cookie config.

Reply:
Done.